presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -150,7 +150,8 @@ public void testColumnName(String columnName)
         }
 
         if (typeName.equals("date") // date gets stored as varchar
-                || typeName.equals("varbinary")) { // TODO (https://github.com/prestosql/presto/issues/3416)
+                || typeName.equals("varbinary") // TODO (https://github.com/prestosql/presto/issues/3416)
+                || (typeName.contains("char") && dataMappingTestSetup.getSampleValueLiteral().contains(" "))) { // TODO: https://github.com/prestosql/presto/issues/3597
```

Comment:
This condition skips `varchar(3)` & `de ` data mapping too, but the issue happens for `char(3)` type.

Reply:
Thanks